indico/modules/search/client/js/components/SearchBox.jsx

```diff
@@ -60,11 +93,12 @@ export default function SearchBox({onSearch, category}) {
         results={options}
         value={keyword}
         showNoResults={false}
+        open={!!keyword.length}
```

Comment:
```suggestion
        open={!!keyword}
```

this should be enough assuming that `keyword` is a string